### PR TITLE
fix: added scrollbar to the autocomplete-search

### PR DESCRIPTION
**Pull request by rajat-mehra05**
Closes: Issue No. #177 

image

Here's the demo:

[rds.webm](https://user-images.githubusercontent.com/35677839/179937999-65ac62ef-927d-4067-b1db-b41d8a3d42d1.webm)

### CODE REVIEW

File: src/task/style.css [hunk 1]
```diff
@@ -242,6 +242,11 @@ footer {
   margin: 0;
 }
 
+#list-items {
+  max-height: 200px;
+  overflow-y: scroll;
```
Comment (DeRaowl) [inline src/task/style.css:247]: What about overflow-x: If I change css property in existing codebase I am seeing scroll in x axis. Can you check

File: src/task/style.css [hunk 2]
```diff
@@ -242,6 +242,11 @@ footer {
   margin: 0;
 }
 
+#list-items {
```
Comment (DeRaowl) [inline src/task/style.css:245]: We already have styles for #list-items can you check regarding this?

File: src/task/style.css [hunk 3]
```diff
@@ -242,6 +242,11 @@ footer {
...
+  max-height: 200px;
```
Comment (DeRaowl) [inline src/task/style.css:246]: Can we make use of relative units like rem

File: src/task/style.css (hunk 1 above)
Comment (rajat-mehra05) [reply src/task/style.css]: Resolved.

File: src/task/style.css (hunk 3 above)
Comment (DeRaowl) [reply src/task/style.css]: LGTM

**Review by Pavangbhat — APPROVED** [review]

**Pull request merged**